cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ExecutableExecutionData.java

```diff
@@ -126,14 +126,22 @@ public void startExecutable(@Param(ScoreLangConstants.EXECUTABLE_INPUTS_KEY) Lis
                     }
                 }
             }
+            if (systemContext.containsKey(ScoreLangConstants.DEBUGGER_FLOW_INPUTS)) {
+
+                Context flowContext = runEnv.getStack().popContext();
+                flowContext.putVariables(debuggerBreakpointsHandler.resolveInputs(systemContext));
+                runEnv.getStack().pushContext(flowContext);
+            }
+
             executableInputs = Collections.unmodifiableList(mutableInputList);
+
             //restore what was already prompted and add newly prompted values
             Map<String, Value> promptedValues = runEnv.removePromptedValues();
             promptedValues.putAll(missingInputHandler.applyPromptInputValues(systemContext, executableInputs));
 
             Map<String, Prompt> promptArguments = runEnv.removePromptArguments();
 
-            if (systemContext.containsKey(ScoreLangConstants.DEBUGGER_INPUTS)) {
+            if (systemContext.containsKey(ScoreLangConstants.DEBUGGER_STEP_INPUTS)) {
```

Comment:
We should change this to `DEBUGGER_EXECUTABLE_INPUTS` everywhere we use it, because it's not really a step input